src/Console/ConsoleInputOption.php

```diff
@@ -90,7 +90,7 @@ class ConsoleInputOption
      * @param string $short The short alias for this option
      * @param string $help The help text for this option
      * @param bool $isBoolean Whether this option is a boolean option. Boolean options don't consume extra tokens
-     * @param string|bool|null $default The default value for this option.
+     * @param mixed $default The default value for this option.
```

Comment:
@markstory Changed to mixed in annotation but removed native type hint. Everything is cast to string.